recipes/exempi/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "exempi" %}
+{% set version = "2.4.2" %}
+{% set sha256 = "5b0db25fa5ab11344adbc26e1d4a3ace2ad1166a7bea4f4d3fd1837e982bc6ec" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.bz2
+  url: https://libopenraw.freedesktop.org/download/{{ name }}-{{ version }}.tar.bz2
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - gcc  # [linux]
+    - boost
+    - expat
+    - zlib
```

Comment:
Please check our pinning [scheme](https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L39-L86) and add the pinned versions here.